cli/command_content_verify.go

```diff
@@ -26,7 +26,9 @@ type commandContentVerify struct {
 }
 
 func (c *commandContentVerify) setup(svc appServices, parent commandParent) {
-	cmd := parent.Command("verify", "Verify that each content is backed by a valid blob")
+	cmd := parent.Command("verify", "Verify that each content is backed by a valid blob. " +
```

Comment:
this is good - we have an example of such lengthy comment in a few places, but I would add an extra empty line after the first line summary:

```
Verify that each content is backed by a valid blob.

Does **not** verify that all blobs for all existing snapshots do exist. 
See also `kopia snapshot verify` and https://kopia.io/docs/advanced/consistency/ for details
```